core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -10501,7 +10508,8 @@ private void checkCustomColumnResolving(String table) {
     final String sql4 = "insert into EMP_MODIFIABLEVIEW2(\"extra\" INTEGER)"
         + " (empno, ename, job, \"extra\")\n"
         + "values (1, 'Arthur', 'clown', true)";
-    s.withSql(sql4).ok();
+    s.withConformance(SqlConformanceEnum.BIG_QUERY).withTypeCoercion(true).withSql(sql4).ok();
```

Comment:
you've checked that it's ok on BigQuery... but if so, why only BigQuery? should it fail on other conformances?

Reply:
I've verified for that for other conformance enums these should also be included:
- MySql 5

It should not be included for :
- Oracle 10 
- Oracle 12
- Strict 2003

I wasn't able to determine it for Presto